### PR TITLE
Small HPA fixes

### PR DESCRIPTION
**Pull request by colin014**
Fix invalid json response and send not found message if needed

### CODE REVIEW

File: api/hpa.go [hunk 1]
```diff
@@ -118,11 +118,6 @@ func DeleteHpaResource(c *gin.Context) {
 func GetHpaResource(c *gin.Context) {
 	scaleTarget, ok := ginutils.RequiredQuery(c, "scaleTarget")
 	if !ok {
-		c.JSON(http.StatusBadRequest, pkgCommmon.ErrorResponse{
```
Comment (bonifaido) [inline api/hpa.go]: Why this block got deleted?

Comment (bonifaido) [reply api/hpa.go]: Sorry I missed that, probably it should have a name like `RequiredQueryOrError` just to make it more understandable.

Comment (colin014) [reply api/hpa.go]: You're right. I will rename this function

Comment (bonifaido) [reply api/hpa.go]: OrAbort is probably more Gin-ish! :)

**Pull request merged**